docker-compose.yml

```diff
@@ -27,3 +27,15 @@ services:
     volumes:
       - $ROOT/demo:/orderly
       - $ROOT/config/orderlyweb:/etc/orderly/web
+  mq:
+    image: redis
+    ports:
+      - "6379:6379"
+  flower:
+    image: mher/flower:0.9.5
+    ports:
+      - "5555:5555"
+    environment:
+      - FLOWER_PORT=5555
+      - CELERY_BROKER_URL=redis://guest@montagu_mq_1//
```

Comment:
Should be able to use `redis://guest@mq//` to avoid hard-coding Compose project name and instance no.

Same for next line.